### PR TITLE
Annotate code 

### PR DESCRIPTION
**Pull request by GerbrichFerdinands**
Add comments to 

- `double.py`, balance strategy (Dynamic Resampling)
- `analysis.py`, (average) Time to Discovery

### CODE REVIEW

File: asreview/analysis/analysis.py [hunk 1]
```diff
@@ -276,7 +276,7 @@ def rrf(self, val=10, x_format="percentage", **kwargs):
         return (None, None, None)
 
     def avg_time_to_discovery(self, result_format="number"):
-        """Get the best/last estimate on how long it takes to find a paper.
+        """Get the best/last estimate on how long it takes to find a paper, the Time to Discovery (TD).
```
Comment (J535D165) [inline asreview/analysis/analysis.py:279]:
```
        """Compute the average Time to Discovery.

        Get the best/last estimate on how long it takes to find a paper, the Time 
        to Discovery (TD).

        Returns
        -------
```

The first line should be short (72char). Longer description below.

File: asreview/balance_strategies/double.py [hunk 2]
```diff
@@ -62,34 +62,39 @@ def __init__(self, a=2.155, alpha=0.94, b=0.789, beta=1.0,
         self._random_state = get_random_state(random_state)
 
     def sample(self, X, y, train_idx, shared):
+        # record ids of already labeled publications, as inclusions (ones) or exclusions (zeroes)
```
Comment (J535D165) [inline asreview/balance_strategies/double.py:65]: Maximum length is 79 characters.

File: asreview/analysis/analysis.py (hunk 1 above)
Comment (GerbrichFerdinands) [reply asreview/analysis/analysis.py]: Thanks for the heads up!

Comment (GerbrichFerdinands) [reply asreview/analysis/analysis.py]: Like this? 
>      """Get the best/last estimate on how long it takes to find a paper.
>         Known as Time to Discovery (TD).

Comment (J535D165) [reply asreview/analysis/analysis.py]: There should be another newline after the first line. Like the format above. 

1. one line description
2. blank line
3. Long description.

Comment (qubixes) [reply asreview/analysis/analysis.py]: I have already changed it ;). Though the long description could still use some work.

Comment (J535D165) [reply asreview/analysis/analysis.py]: Thanks

**Comment by GerbrichFerdinands** [comment]
Thanks for the adjustments @qubixes!

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Thanks folks!

**Pull request merged**